### PR TITLE
Added support for isbn-utils

### PR DESCRIPTION
**Pull request by elhaard**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `npm run new-package package-name`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: isbn-utils/tslint.conf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (zhengbli) [inline isbn-utils/tslint.conf]: This file should be `tslint.json`

File: isbn-utils/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "files": [
+        "index.d.ts",
+        "isbn-utils-tests.ts"
+    ],
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
```
Comment (zhengbli) [inline isbn-utils/tsconfig.json]: Add a `lib` option specifying what libraries should be used with it

File: isbn-utils/tslint.conf (hunk 1 above)
Comment (elhaard) [inline isbn-utils/tslint.conf]: Of course. Fixed. Please squash on merge.

File: isbn-utils/tsconfig.json (hunk 2 above)
Comment (elhaard) [inline isbn-utils/tsconfig.json]: Done

**Comment by elhaard** [comment]
@zhengbli, sorry to bother you again.
I have now fixed the issues, you pointed out. Could you please have another look at this PR?

If you do merge it, please squash the commits on merge.
Or should I squash them myself and open a new PR?

**Comment by zhengbli** [comment]
👍  Thanks @elhaard !

**Pull request merged**